src/scripts/deploy.sh

```diff
@@ -211,6 +211,9 @@ while [ $# -gt 0 ] ; do
     -2 | --tier2-sub-id)
       shift
       subs_args+=("-2 ${1}") ;;
+    -3 | --tier3-sub-id)
```

Comment:
`show_help()` could use this value and a description

Reply:
Added!